tests/test_ext.py

```
@@ -118,3 +119,15 @@ def test_compute_shingled_simhashes() -> None:
         assert len(matches) == 0, f"{i} vs different song"
+
+
+def test_decode_postgres_array() -> None:
+    fp_str = "{" + ",".join(map(str, TEST_1A_FP_RAW)) + "}"
+    fp_array = decode_postgres_array(fp_str)
+    assert fp_array == array.array("i", TEST_1A_FP_RAW)
+
+    assert decode_postgres_array("{}") == array.array("i")
+    assert decode_postgres_array("{1}") == array.array("i", [1])
+
+    with pytest.raises(ValueError):
```

Ruff (0.8.2)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)